packages/edit-site/src/components/global-styles/variations/variations-typography.js

```diff
@@ -34,38 +31,22 @@ export default function TypographyVariations( { title, gap = 2 } ) {
 			>
 				{ typographyVariations &&
 					typographyVariations.length &&
```

Comment:
this can go too, right? we are already checking this before and escaping early

Reply:
Thanks! Done in 1b5d0c341e